src/gui/search/searchwidget.h

```diff
@@ -32,8 +33,8 @@
 #include <QList>
 #include <QPointer>
 #include <QWidget>
-
-class QTabWidget;
+#include <QObject>
+#include <QEvent>
```

Comment:
You should forward declare these classes here and move includes to .cpp.

Reply:
What's the advantage of that? Just curious.

Reply:
To avoid polluting other compilation units with unnecessary indirectly included headers.